.github/ISSUE_TEMPLATE/sub-issue.yml

```diff
@@ -0,0 +1,36 @@
+name: Sub-issue
+description: Describe an idea, problem, or feature that is related to a parent issue.
+
+body:
+  - type: markdown
+    id: title-help
+    attributes:
+      value: |
+        > Titles should be short, descriptive, and compelling. Use sentence case (don't capitalize unnecessarily).
```

Comment:
Can we adjust the label of the "Add a title" field. I'd suggest "Sub-issue title"

Reply:
Are you suggesting we change the `id` value or text in the `value`?

Reply:
I'm not sure with it would be. It's the label for the field which currently says "Add a title" not sure if that's even editable though.

Reply:
Ah. I looked and that doesn't appear to be part of the template; it's GitHub-controlled.